cmd/server/assets/realmadmin/_form_general.html

```diff
@@ -56,6 +56,8 @@
     </small>
   </div>
 
+  {{/*TODO(whaught): knock this out at v0.21.0*/}}
+  {{if .devMode}}
```

Comment:
We should really control this with a more specific configuration value, but this is probably fine for now.